site/_posts/2020-10-26-announcing-fusionauth-1-20.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: blog-post
+title: Announcing FusionAuth 1.20
+description: The FusionAuth 1.20 Release offers more forms, SAML improvements, docker changes and more.
+author: Dan Moore
+image: blogs/release-1-20/fusionauth-release-1-20.png
+category: blog
+excerpt_separator: "<!--more-->"
+---
+
+We're excited to announce the release of version 1.20. The 1.20 release shipped on Oct 26, 2020. This version delivers new features as well as resolving issues for users on version 1.19 and older.
+
+<!--more-->
+
+## Highlights
+
+In addition to bug fixes and user interface improvements, there are a couple of new features and improvements which deserve a spotlight.
+
+### SAML, SAML, SAML
+
+With this release, FusionAuth implements additional parts of the SAML 2.0 standard.
+
+FusionAuth now has support for SAML POST bindings when it is acting as either the SP or the IdP. This allows FusionAuth to integrate with a number of popular commercial off the shelf applications and websites, including [login.gov](https://developers.login.gov/saml/) and [Alfresco](https://docs.alfresco.com/saml/concepts/saml-overview.html).
+
+FusionAuth also supports the `SessionIndex` attribute, which is optional, but required by some software applications, notably Artifactory.
+
+Finally FusionAuth now honors the `AssertionConsumerServiceURL` in a SAML request. Multiple redirect URLs can be configured in the application SAML settings as well.
+
+### Docker changes
+
+In this release, we updated the base image for Docker from `alpine` to `ubuntu:focal`. This should not impact functionality, but it's worth paying attention to if you are building Docker images based on our image.
+
+We made this change because in order to run on `alpine` without including the GNU C Library (`glibc`) we had to use a custom build of OpenJDK compiled using the `musl` C library. Due to some possible performance concerns with this option, we have moved to an official build of JDK provided by AdoptOpenJDK. This build is compiled using `glibc`. Using `ubuntu:focal` increased the image size by approximately 30 MB compared to the `alpine` image, unfortunately. However, until we can obtain official builds from AdoptOpenJDK based upon the `musl` C library, we are not planning to ship an official FusionAuth image on `alpine` due to the performance concerns.
```

Comment:
Unfortunately, using `ubuntu:focal` increased the image size by approximately 30 MB compared to the `alpine` image.